_posts/2024/2024-02-06-our-process.md

```diff
@@ -0,0 +1,453 @@
+Back in 2022 I wrote that there are two ways of looking at roles in a small-to-medium tech company:
+what people do, and [what risks they address][mitigation].
+Coincidentally,
+I had just started working with someone who has turned out to be
+[the best product manager I've ever met][dottie].
+In the two years since she joined the company
+she has overseen lots of improvements in our software engineering processes
+(partly, I suspect, to make it easier for her to do her job properly).
+She recently summarized the state of play to orient a new hire;
+what follows is a summary of her summary with a few explanatory notes
+that I hope will be useful to other software developers
+working in research-intensive organizations.
+
+I believe the most important part of what follows
+is how she has organized her thinking.
+Each process exists *to answer key questions*,
+has its own *frequency*,
+is *someone's responsibility*,
+and generates *specific products*.
+If you have a description like this of how your team operates,
+please [get in touch](mailto:{{site.author.email}}):
+I think we could learn a lot from comparisons.
+
+## Company Goals and Priority Projects
+
+This activity happens up in the stratosphere.
+As an engineering manager I may occasionally be called on for input,
+but for the most part this is the ground for everything else described below.
+
+-   Summary: Prioritized company objectives and initiatives for the year
+-   Key Questions:
+    -   What should we invest in/focus on?
+-   Frequency: 6-24 months
+-   Responsible: executive team 
+-   Contributors: company leadership and technical experts at all levels
+-   Products:
```

Comment:
These don't feel like the right "products" here. The product would be a prioritized list of strategic focus areas and investments for the company. The MBR is a way to track progress towards this. The All hands would be one way to communicate the progress and/or changes to the goals